### ISSUE TITLE
[insserver] GLIBC 2.15 issues in recent update

### ISSUE THREAD

**Issue by jargon-au** [GameServerManagers/LinuxGSM#324]
Hi Daniel, 

Insurgency now requires the following fix in order to launch on CentOS 6 and probably other distros in the same basket:

export LD_LIBRARY_PATH=/home/insserver/serverfiles/bin <-- substitute own path to bin folder here

Followed by:
http://danielgibbs.co.uk/2014/08/garrys-mod-linux-server-glibc_2-15/

**Comment by hyperbolic2346** [comment]
I have this issue with my docker container running ubuntu:latest.

**Comment by Pringlez** [comment]
Same here on Ubuntu 12.04.5 LTS, can't launch the server.

**Comment by hyperbolic2346** [comment]
You should be able to launch the server if you export the path shown in the original comment. It depends on where you have your server installed, but just export that bin directory and you should be good to go.

**Comment by Scarsz** [comment]
I can confirm this on Ubuntu 14.10 x64

Error probably the suspect:
`./srcds_linux: error while loading shared libraries: libtier0.so: cannot open shared object file: No such file or directory`

**Comment by jargon-au** [comment]
Ah yes, posting the error as well as the solution would have helped, my bad :P

**Comment by dgibbs64** [comment]
After looking at this a hotfix should resolve the issue. There is already a GLIBC fix in place. I don't believe any action will be required on my part. However can someone please confirm this for me.

**Comment by Roversword** [comment]
Hi all

I tried to install an insurgency server the first time today (on Ubuntu 14.04.2). Install went fine, but starting does not work.
I get the ominous error message:
./srcds_linux: error while loading shared libraries: libtier0.so: cannot open shared object file: No such file or directory

So far I have found indications that this is related to the GLIBC issue, however, I am a little confused.
According to ldd --version the Ubuntu 14.04.2 install runs EGLIBC 2.19.

So version 2.15 is exact requirement or is it the minimum requirement? Because if it is the latter, then Ubuntu 14.04.2 should work out of the box, shouldn't it?
